### PR TITLE
fix(web/applications): fixed issue not allowing to re-upload a file (BOAS-623)

### PR DESCRIPTION
**Pull request by francescobrenci-AR**
fixed issue not allowing to re-upload a file

## Type of change 🧩

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/server/views/applications/case-documentation/documentation-folder.njk [hunk 1]
```diff
@@ -61,10 +61,11 @@
 					<hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
 					{% if documentationFiles |length %}
 						<form class="pins-files-list" method="post">
+							<caption class="display--sr-only">Manage Documents in this location</caption>
```
Comment (lutamKC) [inline apps/web/src/server/views/applications/case-documentation/documentation-folder.njk:64]: this happens on both chrome and firefox. apparently the browser removes the tag caption which is supposed to exist only inside a <table>

**Review by lutamKC — CHANGES_REQUESTED** [review]

File: apps/web/src/server/views/applications/case-documentation/documentation-folder.njk (hunk 1 above)
Comment (francescobrenci-AR) [reply apps/web/src/server/views/applications/case-documentation/documentation-folder.njk]: done

**Review by lutamKC — DISMISSED** [review]

**Review by lutamKC — APPROVED** [review]

**Pull request merged**